icu4c/APIChangeReport.md

```diff
@@ -0,0 +1,1094 @@
+| localematcher.h | Builder&amp; icu::LocaleMatcher::Builder::setFavorSubtag(ULocMatchFavorSubtag) |  (missing) | DraftICU 65
+| localematcher.h | Builder&amp; icu::LocaleMatcher::Builder::setSupportedLocales(Iter, Iter) |  (missing) | DraftICU 65
+| localematcher.h | Builder&amp; icu::LocaleMatcher::Builder::setSupportedLocales(Locale::Iterator&amp;) |  (missing) | DraftICU 65
+| localematcher.h | Builder&amp; icu::LocaleMatcher::Builder::setSupportedLocalesFromListString(StringPiece) |  (missing) | DraftICU 65
+| localematcher.h | Builder&amp; icu::LocaleMatcher::Builder::setSupportedLocalesViaConverter(Iter, Iter, Conv) |  (missing) | DraftICU 65
+| localematcher.h | Locale icu::LocaleMatcher::Result::makeResolvedLocale(UErrorCode&amp;) const |  (missing) | DraftICU 65
+| localematcher.h | LocaleMatcher icu::LocaleMatcher::Builder::build(UErrorCode&amp;) const |  (missing) | DraftICU 65
+| localematcher.h | LocaleMatcher&amp; icu::LocaleMatcher::operator=(LocaleMatcher&amp;&amp;) |  (missing) | DraftICU 65
+| localematcher.h | Result icu::LocaleMatcher::getBestMatchResult(Locale::Iterator&amp;, UErrorCode&amp;) const |  (missing) | DraftICU 65
+| localematcher.h | Result icu::LocaleMatcher::getBestMatchResult(const Locale&amp;, UErrorCode&amp;) const |  (missing) | DraftICU 65
+| localematcher.h | Result&amp; icu::LocaleMatcher::Result::operator=(Result&amp;&amp;) |  (missing) | DraftICU 65
+| localematcher.h | UBool icu::LocaleMatcher::Builder::copyErrorTo(UErrorCode&amp;) const |  (missing) | DraftICU 65
+| localematcher.h | const Locale* icu::LocaleMatcher::Result::getDesiredLocale() const |  (missing) | DraftICU 65
+| localematcher.h | const Locale* icu::LocaleMatcher::Result::getSupportedLocale() const |  (missing) | DraftICU 65
+| localematcher.h | const Locale* icu::LocaleMatcher::getBestMatch(Locale::Iterator&amp;, UErrorCode&amp;) const |  (missing) | DraftICU 65
+| localematcher.h | const Locale* icu::LocaleMatcher::getBestMatch(const Locale&amp;, UErrorCode&amp;) const |  (missing) | DraftICU 65
+| localematcher.h | const Locale* icu::LocaleMatcher::getBestMatchForListString(StringPiece, UErrorCode&amp;) const |  (missing) | DraftICU 65
+| localematcher.h | double icu::LocaleMatcher::internalMatch(const Locale&amp;, const Locale&amp;, UErrorCode&amp;) const |  (missing) | Internal
+| localematcher.h | <tt>enum</tt> ULocMatchDemotion::ULOCMATCH_DEMOTION_NONE |  (missing) | DraftICU 65
+| localematcher.h | <tt>enum</tt> ULocMatchDemotion::ULOCMATCH_DEMOTION_REGION |  (missing) | DraftICU 65
+| localematcher.h | <tt>enum</tt> ULocMatchFavorSubtag::ULOCMATCH_FAVOR_LANGUAGE |  (missing) | DraftICU 65
+| localematcher.h | <tt>enum</tt> ULocMatchFavorSubtag::ULOCMATCH_FAVOR_SCRIPT |  (missing) | DraftICU 65
+| localematcher.h | icu::LocaleMatcher::Builder::Builder() |  (missing) | DraftICU 65
+| localematcher.h | icu::Locale
```

Comment:
Should be promoted.

Reply:
> Should be promoted.


LocaleMatcher is draft ICU 65 - it should be promoted?

Reply:
No. Sorry, I seem to have put the comment on the wrong line :-(

Reply:
On my machine the comment shows on `void icu::DecimalFormat::setMultiplierScale(int32_t)` and `int32_t icu::DecimalFormat::getMultiplierScale() const` not on LocaleMatcher.

Reply:
I see:

<img width="701" alt="image" src="https://user-images.githubusercontent.com/855219/64573130-09994f80-d31f-11e9-898b-c985479376bc.png">

but, I will take a look at the `MultiplierScale` items.

Reply:
![image](https://user-images.githubusercontent.com/1145762/64573354-e3c07a80-d31f-11e9-956f-600a655644fc.png)

+1 for promoting the DecimalFormat method.  Note that ICU 65 should promote *both* ICU 62 *and* ICU 63, since ICU 62 was the weird summer release.

Reply:
… Promoted.

Reply:
@sffc I think the two fieldPosition items are the only ICU 62  drafts remaining.